lib/wrappers/python/emulator_contourwall.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import cv2
+import time
+
+class EmulatorContourWall:
+    def __init__(self):
+        self.__cv_window_name = "ContourWall Emulation"
+        self.rows = 40
+        self.cols = 60
+        self.cell_size = 10
+        self.pushed_frames: int = 0
+        self.matrix = np.zeros((self.rows * self.cell_size, self.cols * self.cell_size, 3), dtype=np.uint8) 
```

Comment:
The field `self.matrix`, should be made private as you do not want people to use it. You do that by adding two underscores in front of its name: `__`

```python
self.__matrix = np.zeros((self.rows * self.cell_size, self.cols * self.cell_size, 3), dtype=np.uint8)
````